rosdep/python.yaml

```diff
@@ -3087,6 +3087,27 @@ python-scipy:
     yakkety_python3: [python3-scipy]
     zesty: [python-scipy]
     zesty_python3: [python3-scipy]
+python-seaborn:
+  arch: [python2-seaborn]
+  debian: [python-seaborn]
+  fedora:
+    pip:
+      packages: [seaborn]
+  gentoo: [dev-python/seaborn]
+  ubuntu:
+    artful: [python-seaborn]
```

Comment:
It would make sense to use the wildcard format for ubuntu
```
'*': [python-seaborn]
'trusty':
  pip:
    packages: [seaborn]
```

It's more compact and future proof.